Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/storage/FamilyTestStorage.kt

```diff
@@ -23,10 +23,18 @@ class FamilyTestStorage @Inject constructor(
         dao.insert(test.toEntity())
     }
 
-    suspend fun update(identifier: TestIdentifier, update: (FamilyCoronaTest) -> FamilyCoronaTest) {
+    suspend fun update(identifier: TestIdentifier, update: suspend (FamilyCoronaTest) -> FamilyCoronaTest) {
         dao.update(identifier, update)
     }
 
+    suspend fun updateAll(identifiers: Set<TestIdentifier>, update: suspend (FamilyCoronaTest) -> FamilyCoronaTest) {
+        dao.updateAll(identifiers, update)
+    }
+
+    suspend fun updateAll(tests: List<FamilyCoronaTest>) {
```

Comment:
Not a fan of this. See comment above. This can be dangerous

Reply:
what do you suggest?

Reply:
I added a list update here https://github.com/corona-warn-app/cwa-app-android/pull/5018 that can be used to move all items into the recycle bin

Reply:
ok, reverted all